docker-app/qfieldcloud/urls.py

```diff
@@ -54,9 +54,9 @@
     ),
     path("docs/", schema_view.with_ui("redoc", cache_timeout=0), name="schema-redoc"),
     path("admin/", admin.site.urls),
-    path("api/v1/auth/registration/", include("rest_auth.registration.urls")),
-    path("api/v1/auth/token/", auth_views.AuthTokenView.as_view()),
-    path("api/v1/auth/", include("rest_auth.urls")),
+    path("api/v1/auth/login/", auth_views.LoginView.as_view()),
+    path("api/v1/auth/token/", auth_views.LoginView.as_view()),
```

Comment:
Is it expected that the token view is the same as loginview ? Does that mean that to refresh the token, the app must send (and thus store) the login ?

Don't we need an endpoint to just get a new token, using the existing one ?

Reply:
We always had it like this, which is wrong in many ways. It can be addressed in another PR.